lib/elixir/lib/file.ex

```diff
@@ -256,6 +256,14 @@ defmodule File do
   Missing parent directories are not created.
   Returns `:ok` if successful, or `{:error, reason}` if an error occurs.
 
+  ## Examples
```

Comment:
Can you please move the examples to the bottom? Or after the "Typical error reasons are:"? That's because the error reasons are part of the text not of the example, thank you.